Dockerfile

```diff
@@ -30,10 +41,25 @@ RUN chmod 644 /etc/systemd/system/iptables_setup.service
 # connector configuration to handle initial boot and disconnections
 COPY ./connector.autoload /etc/openvpn3/autoload/connector.autoload
 
+# Download the prometheus script exporter to expose OpenVPN connector metrics
```

Comment:
These additional steps add a small Prometheus exporter service that can expose OpenVPN Connector metrics to Prometheus, allowing monitoring and alerting for OpenVPN Cloud Connectors.